### PR TITLE
Update README.md to reflect latest developments

### PR DESCRIPTION
**Pull request by taymonbeal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,13 +22,13 @@ data.
...
-  e.g., `<iframe src="https://fledge-shim.example/v/1234">`.
+  e.g., `<iframe src="https://fledge-shim.example/0.1.html">`.
 
 - A _library_ that consumers use to communicate with the frame over
   `postMessage`.
 
-Almost all of the work happens in the frame; the library is a small adapter
-that translate the API from functions to messages.
+Almost all of the work happens in the frame; the library is a small adapter that
+translate the API from functions to messages.
```
Comment (jeffkaufman) [inline README.md:31]: translate -> translates

File: README.md [hunk 2]
```diff
@@ -136,10 +139,11 @@ minimally useful as early as possible. Our current planned stages are:
 Implement core functionality:
```
Comment (jeffkaufman) [inline README.md:139]: Do you want to say that V1 is complete?

File: .prettierrc.json [hunk 3]
```diff
@@ -1 +1 @@
-{ "quoteProps": "preserve" }
+{ "quoteProps": "preserve", "proseWrap": "always" }
```
Comment (jeffkaufman) [inline .prettierrc.json:1]: for the future: it would have been better to separate re-wrapping and content changes into separate PRs

**Review by jeffkaufman — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (taymonbeal) [reply README.md:139]: It's not quite yet, there are still some things missing like only authorizing particular buyers on a given auction. I'm planning on soon going through the WICG proposal and filing an issue for each API feature that hasn't been implemented yet, with an appropriate milestone attached to each one.

**Pull request merged**